### PR TITLE
[fix]: Build crash due to location permissions

### PR DESCRIPTION
**Pull request by utkarsh006**
## Description
- Solves #43 

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

## Screenshots / Screen Recordings

<details><summary><h3>Before Fix</h3></summary>

https://github.com/user-attachments/assets/73d41655-e5f3-4ca4-b6ab-bf041ffee7fd

</details>

<details><summary><h3>After Fix</h3></summary>

https://github.com/user-attachments/assets/ddfb42fe-e32d-444b-b8eb-d5b326974b05

</details>

### CODE REVIEW

**Comment by utkarsh006** [comment]
@Vishwajith-Shettigar please review

**Comment by Vishwajith-Shettigar** [comment]
@utkarsh006 please create issue before creating PR to fix particular issue.

**Comment by utkarsh006** [comment]
@Vishwajith-Shettigar please check now :)

**Comment by Vishwajith-Shettigar** [comment]
Can you please add before and after screen recordings, that shows your solution resolved the issue.

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
```
Comment (Vishwajith-Shettigar) [inline .idea/misc.xml:1]: Please delete .idea folder.

Comment (utkarsh006) [reply .idea/misc.xml]: done @Vishwajith-Shettigar

**Comment by Vishwajith-Shettigar** [comment]
Thanks @utkarsh006.

**Comment by utkarsh006** [comment]
@Vishwajith-Shettigar Your welcome. Please include this PR as per [Hacktoberfest Guidelines](https://hacktoberfest.com/participation/#maintainers)

**Pull request merged**